tests/modules/humann/test.yml

```diff
@@ -0,0 +1,7 @@
+- name: humann test_humann
+  command: nextflow run tests/modules/humann -entry test_humann -c tests/config/nextflow.config
+  tags:
+    - humann
+  files:
+    - path: output/humann/versions.yml
```

Comment:
Remove versions.yml from the file, if you update to the latest version of `nf-core` you won't have to delete these manually (you can also see that you are not getting any other output, is this supposed to happen?)